### PR TITLE
Nfdiv 1610

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1610

### Change description ###
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/functional/features/joint-conditional-order.feature [hunk 1]
```diff
@@ -46,3 +46,16 @@ Feature: Joint conditional order
...
+  Scenario: Applicants CO entitlement granted
+    Given I set the case state to "AwaitingPronouncement"
+    And a case worker updates court case hearing
+    When I click "Sign out"
+    And I login with applicant "1"
+    Then the page URL should be "/hub-page"
+    And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 6:46PM."
+    When I click "Sign out"
+    And I login with applicant "2"
+    Given I go to "/"
```
Comment (benouaer) [inline src/test/functional/features/joint-conditional-order.feature:59]: this should probably be a 'When'

File: src/main/steps/respondent/hub-page/content.ts [hunk 2]
```diff
@@ -79,9 +79,9 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
   },
   conditionalOrderPronounced: {
     line1: `You have been granted a ‘conditional order’ by the court. Your conditional order was formally pronounced
-    (read out) by a judge at ${userCase.coCourt === ConditionalOrderCourt.BIRMINGHAM ? birmingham : buryStEdmunds} on ${
-      userCase.coDateOfHearing
-    }.
+    (read out) by a judge at ${
+      userCase.coCourt === ConditionalOrderCourt.BIRMINGHAM ? birmingham : buryStEdmunds
```
Comment (benouaer) [inline src/main/steps/respondent/hub-page/content.ts:83]: is this needed? wouldn't just using the enum work? or if not toString()

**Review by benouaer — APPROVED** [review]

File: src/main/steps/respondent/hub-page/content.ts (hunk 2 above)
Comment (velimd) [reply src/main/steps/respondent/hub-page/content.ts:83]: nope, as the enums are more like individual variables

**Review by lara-mpt — COMMENTED** [review]
Looks like the progress bar needs updating?

**Pull request merged**